### PR TITLE
FX2024 Domain & Capabilities update / Optimize Cloud Usage domain

### PR DESCRIPTION
**Pull request by arthurclares**
## 🛠️ Description
This PR updates all the existing capabilities that are part of the **optimize cloud usage** domain according to the new FinOps framework.

It does not contain any new capability. This PR only covers what needs to be renamed and removed. New capabilities will be covered by a future PR.

### 🔬 How did you test this change?

> - [ ] 🤏 Lint tests
> - [ ] 🤞 PS -WhatIf / az validate
> - [ ] 👍 Manually deployed + verified
> - [ ] 💪 Unit tests
> - [ ] 🙌 Integration tests

### 🙋‍♀️ Do any of the following that apply?

> - [ ] 🚨 This is a breaking change.
> - [ ] 🤏 The change is less than 20 lines of code.

### 📑 Did you update `docs/changelog.md`?

> - [ ] ✅ Yes (required for `dev` PRs)
> - [x] ➡️ Will cover in a future PR (feature branch PRs only)
> - [ ] ❎ Not needed (small/internal change)

### 📖 Did you update documentation?

> - [x] ✅ Public docs in `docs` (required for `dev`)
> - [ ] ✅ Internal dev docs in `src` (required for `dev`)
> - [ ] ➡️ Will cover in a future PR (feature branch PRs only)
> - [ ] ❎ Not needed (small/internal change)

### CODE REVIEW

File: docs/_docs/framework/capabilities/optimize/rate-optimization.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ This article helps you understand the managing commitment-based discounts capabi
 ---
 
 <a name="definition"></a>
-**Managing commitment-based discounts is the practice of obtaining reduced rates on cloud services by committing to a certain level of usage or spend over a specific period.**
+**Rate otpimization is the practice of obtaining reduced rates on cloud services by committing to a certain level of usage or spend over a specific period.**
```
Comment (flanakin) [inline docs/_docs/framework/capabilities/optimize/rate-optimization.md:36]:
```suggestion
**Rate otpimization is the practice of obtaining reduced rates on cloud services, often by committing to a certain level of usage or spend over a specific period.**
```

File: docs/_docs/framework/capabilities/optimize/rate-optimization.md [hunk 2]
```diff
@@ -80,12 +80,13 @@ At this point, you have commitment-based discounts in place. As you move beyond
...
 
 <br>
 
 ## 🍎 Learn more at the FinOps Foundation
 
-This capability is a part of the FinOps Framework by the FinOps Foundation, a non-profit organization dedicated to advancing cloud cost management and optimization. For more information about FinOps, including useful playbooks, training and certification programs, and more, see the [Managing commitment-based discounts capability](https://www.finops.org/framework/capabilities/manage-commitment-based-discounts/) article in the FinOps Framework documentation.
+This capability is a part of the FinOps Framework by the FinOps Foundation, a non-profit organization dedicated to advancing cloud cost management and optimization. For more information about FinOps, including useful playbooks, training and certification programs, and more, see the [Rate Optimization](https://www.finops.org/framework/capabilities/rate-optimization/) article in the FinOps Framework documentation.
```
Comment (flanakin) [inline docs/_docs/framework/capabilities/optimize/rate-optimization.md:89]:
```suggestion
This capability is a part of the FinOps Framework by the FinOps Foundation, a non-profit organization dedicated to advancing cloud cost management and optimization. For more information about FinOps, including useful playbooks, training and certification programs, and more, see the [Rate optimization](https://www.finops.org/framework/capabilities/rate-optimization/) article in the FinOps Framework documentation.
```

File: docs/_docs/framework/capabilities/optimize/rate-optimization.md [hunk 3]
```diff
@@ -80,12 +80,13 @@ At this point, you have commitment-based discounts in place. As you move beyond
   - Consider implementing a consistent utilization and coverage monitoring system that covers all accounts.
 - Establish a process for centralized purchasing of commitment-based offers, assigning responsibility to a dedicated team or individual.
 - Consider programmatically aligning governance policies with commitments to prioritize SKUs and locations that are covered by reservations and aren't fully utilized when deploying new applications.
+- If you need to monitor the usage of commitment discounts outside of the Azure portal, consider deploying FinOps hubs which includes a [Commitment discount dashboard](https://microsoft.github.io/finops-toolkit/power-bi) which summarizes existing and potential savings from commitment discounts, like reservations and savings plans.
```
Comment (flanakin) [inline docs/_docs/framework/capabilities/optimize/rate-optimization.md:83]:
```suggestion
- If you need to monitor the usage of commitment discounts outside of the Azure portal, consider deploying FinOps hubs which includes a [Commitment discount dashboard](../../../../_reporting/power-bi/commitment-discounts.md) which summarizes existing and potential savings from commitment discounts, like reservations and savings plans. 
```

File: docs/_docs/framework/capabilities/optimize/utilization-efficiency.md [hunk 4]
```diff
@@ -89,7 +88,7 @@ You can also find related videos on the FinOps Foundation YouTube channel:
 
 Related FinOps capabilities:
 
-- [Managing commitment-based discounts](./commitment-discounts.md)
+- [Managing commitment-based discounts](./rate-optimization.md)
```
Comment (flanakin) [inline docs/_docs/framework/capabilities/optimize/utilization-efficiency.md:91]:
```suggestion
- [Rate optimization](./rate-optimization.md)
```

File: docs/_docs/framework/capabilities/optimize/utilization-efficiency.md [hunk 5]
```diff
@@ -50,7 +50,7 @@ When you first start managing cost in the cloud, you use the native tools to dri
 - Review and implement [Azure Advisor cost recommendations](https://learn.microsoft.com/azure/advisor/advisor-reference-cost-recommendations).
   - Azure Advisor gives you high-confidence recommendations based on your usage. Azure Advisor is always the best place to start when looking to optimize any workload.
   - Consider [subscribing to Azure Advisor alerts](https://learn.microsoft.com/azure/advisor/advisor-alerts-portal) to get notified when there are new cost recommendations.
-- Review your usage and purchase [commitment-based discounts](./commitment-discounts.md) when it makes sense.
+- Review your usage and purchase [commitment-based discounts](./rate-optimization.md) when it makes sense.
```
Comment (flanakin) [inline docs/_docs/framework/capabilities/optimize/utilization-efficiency.md:53]:
```suggestion
- Review your usage and purchase [Rate optimization](./rate-optimization.md) when it makes sense.
```

File: docs/_docs/framework/capabilities/optimize/utilization-efficiency.md [hunk 6]
```diff
@@ -68,8 +68,7 @@ When you first start managing cost in the cloud, you use the native tools to dri
 
 At this point, you've implemented all the basic cost optimization recommendations and tuned applications to meet the most fundamental best practices. As you move beyond the basics, consider the following points:
 
-- Automate cost recommendations using [Azure Resource Graph](https://learn.microsoft.com/azure/advisor/resource-graph-samples)
-- Implement the [Workload management and automation capability](./workloads.md) for more optimizations.
+
```
Comment (flanakin) [inline docs/_docs/framework/capabilities/optimize/utilization-efficiency.md:71]: Is there a reason to get rid of the ARG link?
```suggestion
- Automate cost recommendations using [Azure Resource Graph](https://learn.microsoft.com/azure/advisor/resource-graph-samples)
```

File: docs/_docs/framework/capabilities/quantify/forecasting.md [hunk 7]
```diff
@@ -137,7 +137,7 @@ You can also find related videos on the FinOps Foundation YouTube channel:
 ## ⏩ Next steps
 
 - [Budget management](./budgeting.md)
-- [Managing commitment-based discounts](../optimize/commitment-discounts.md)
+- [Managing commitment-based discounts](../optimize/rate-optimization.md)
```
Comment (flanakin) [inline docs/_docs/framework/capabilities/quantify/forecasting.md:140]:
```suggestion
- [Rate optimization](../optimize/rate-optimization.md)
```

File: docs/_docs/framework/capabilities/understand/shared-cost.md [hunk 8]
```diff
@@ -45,7 +45,7 @@ Effectively managing shared costs reduces overhead, increases transparency and a
 
 Before you start, it's important to have a clear understanding of your organization's goals and priorities when it comes to managing shared costs. Keep in mind that not all shared costs may need to be redistributed, and some are more effectively managed with other means. Carefully evaluate each shared cost to determine the most appropriate approach for your organization.
 
-This guide doesn't cover commitment-based discounts, like reservations and savings plans. For details about how to handle showback and chargeback, refer to [Managing commitment-based discounts](../optimize/commitment-discounts.md).
+This guide doesn't cover commitment-based discounts, like reservations and savings plans. For details about how to handle showback and chargeback, refer to [Managing commitment-based discounts](../optimize/rate-optimization.md).
```
Comment (flanakin) [inline docs/_docs/framework/capabilities/understand/shared-cost.md:48]:
```suggestion
This guide doesn't cover commitment-based discounts, like reservations and savings plans. For details about how to handle showback and chargeback, refer to [Rate optimization](../optimize/rate-optimization.md).
```

File: docs/_docs/framework/capabilities/optimize/workloads.tobedeleted
Comment (flanakin) [inline docs/_docs/framework/capabilities/optimize/workloads.tobedeleted:1]: Can we merge this and the utilization-efficiency.md file together into workloads.md?

**Review by flanakin — CHANGES_REQUESTED** [review]

**Review by flanakin — APPROVED** [review]

File: docs/_docs/framework/capabilities/optimize/workloads.tobedeleted [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+Review resource usage patterns and determine if they can be scaled down or even shutdown (to stop billing) during off-peak hours. Consider cheaper alternatives to reduce costs.
+
+An effective workload optimization plan can significantly reduce costs by adjusting configuration to match supply to demand dynamically, ensuring the most effective utilization.
+
+<br>
+
+## ▶️ Getting started
+
+When you first start working with a service and managing costs in the cloud, prioritize leveraging native tools within the portal to drive efficiency and optimize costs.
+
```
Comment (KevDLR) [inline docs/_docs/framework/capabilities/optimize/workloads.tobedeleted:51]: Should we mention to use the Advisor Cost Optimization Workbook in this section? 

Use and customize the [Cost optimization workbook](https://learn.microsoft.com/en-us/cloud-computing/finops/cost-optimization-workbook). The Cost Optimization workbook is a central point for some of the most often used tools that can help achieve utilization and efficiency goals.

File: docs/_docs/framework/capabilities/optimize/workloads.tobedeleted [hunk 10]
```diff
@@ -0,0 +1,123 @@
...
+- Does the service support autoscaling?
+  - If the service supports [autoscaling](https://learn.microsoft.com/azure/architecture/best-practices/auto-scaling), configure it to scale based on your application's needs.
+  - Autoscaling can work with autostop behavior for maximum efficiency.
+- Consider automatically stopping and manually starting nonproduction resources during work hours to avoid unnecessary costs.
+  - Avoid automatically starting nonproduction resources that aren't used every day.
+  - If you choose to autostart, be aware of vacations and holidays where resources may get started automatically but not be used.
+  - Consider tagging manually stopped resources. [Save a query in Azure Resource Graph](https://learn.microsoft.com/azure/governance/resource-graph/first-query-portal) or a view in the All resources list and pin it to the Azure portal dashboard to ensure all resources are stopped.
+- Consider architectural models such as containers and serverless to only use resources when they're needed, and to drive maximum efficiency in key services.
+
```
Comment (KevDLR) [inline docs/_docs/framework/capabilities/optimize/workloads.tobedeleted]: Should we highlight to take advantage of the Azure Dev/Test offer? 

•	Leverage the [Azure Dev/Test](https://azure.microsoft.com/en-us/pricing/offers/ms-azr-0148p/) offer that comes with a Visual Studio subscription to take advantage of Azure monthly credits to explore and try various Azure services, benefit from discounted Azure dev/test rates, and enable cost-efficient developing and testing. Although rate optimization strategies can be applied to resources in a development environment, the Azure Dev/Test environment is primarily used for learning and training, development and testing, evaluating proof of concepts, and experimenting and innovating to ensure efficient use of resources.

File: docs/_docs/framework/capabilities/optimize/rate-optimization.md [hunk 11]
```diff
@@ -80,12 +80,13 @@ At this point, you have commitment-based discounts in place. As you move beyond
   - Consider implementing a consistent utilization and coverage monitoring system that covers all accounts.
 - Establish a process for centralized purchasing of commitment-based offers, assigning responsibility to a dedicated team or individual.
 - Consider programmatically aligning governance policies with commitments to prioritize SKUs and locations that are covered by reservations and aren't fully utilized when deploying new applications.
+- If you need to monitor the usage of commitment discounts outside of the Azure portal, consider deploying FinOps hubs which includes a [Commitment discount dashboard](../../../../_reporting/power-bi/commitment-discounts.md) which summarizes existing and potential savings from commitment discounts, like reservations and savings plans.
```
Comment (flanakin) [inline docs/_docs/framework/capabilities/optimize/rate-optimization.md:83]:
```suggestion
- If you need to monitor the usage of commitment discounts outside of the Azure portal, consider deploying FinOps hubs which includes a [Commitment discounts report](../../../../_reporting/power-bi/commitment-discounts.md) that summarizes existing and potential savings from commitment discounts, like reservations and savings plans.
```

File: docs/_docs/framework/capabilities/optimize/utilization-efficiency.md [hunk 12]
```diff
@@ -50,7 +50,7 @@ When you first start managing cost in the cloud, you use the native tools to dri
 - Review and implement [Azure Advisor cost recommendations](https://learn.microsoft.com/azure/advisor/advisor-reference-cost-recommendations).
   - Azure Advisor gives you high-confidence recommendations based on your usage. Azure Advisor is always the best place to start when looking to optimize any workload.
   - Consider [subscribing to Azure Advisor alerts](https://learn.microsoft.com/azure/advisor/advisor-alerts-portal) to get notified when there are new cost recommendations.
-- Review your usage and purchase [commitment-based discounts](./commitment-discounts.md) when it makes sense.
+- Review your usage and purchase [Rate optimization](./rate-optimization.md) when it makes sense.
```
Comment (flanakin) [inline docs/_docs/framework/capabilities/optimize/utilization-efficiency.md:53]:
```suggestion
- Review your usage and purchase [commitment discounts](./rate-optimization.md) when it makes sense.
```

File: docs/_docs/framework/capabilities/optimize/workloads.tobedeleted [hunk 13]
```diff
@@ -32,27 +32,40 @@ This article helps you understand the workload management and automation capabil
 ---
 
 <a name="definition"></a>
-**Workload management and automation refers to running resources only when necessary and at the level or capacity needed for the active workload.**
+**Workload optimization capability involves the analysis and fine-tuning of cloud resources to align with unique usage patterns, prioritizing efficiency and ensuring that workloads deliver optimal business value relative to their cost.**
```
Comment (flanakin) [inline docs/_docs/framework/capabilities/optimize/workloads.tobedeleted:35]:
```suggestion
**Workload optimization capability involves the analysis and fine-tuning of cloud resources to align with unique usage patterns, prioritizing efficiency, and ensuring that workloads deliver optimal business value relative to their cost.**
```

File: docs/_docs/framework/capabilities/optimize/workloads.tobedeleted [hunk 14]
```diff
@@ -32,27 +32,40 @@ This article helps you understand the workload management and automation capabil
...
+Review resource usage patterns and determine if they can be scaled down or even shutdown (to stop billing) during off-peak hours. Consider cheaper alternatives to reduce costs.
+
+An effective workload optimization plan can significantly reduce costs by adjusting configuration to match supply to demand dynamically, ensuring the most effective utilization.
 
 <br>
 
 ## ▶️ Getting started
 
-When you first start working with a service, consider the following points:
+When you first start working with a service and managing costs in the cloud, prioritize leveraging native tools within the portal to drive efficiency and optimize costs.
```
Comment (flanakin) [inline docs/_docs/framework/capabilities/optimize/workloads.tobedeleted:50]:
```suggestion
When you first start working with a service or managing costs in the cloud, prioritize leveraging native tools within the portal to drive efficiency and optimize costs.
```

File: docs/_docs/framework/capabilities/optimize/workloads.tobedeleted [hunk 15]
```diff
@@ -32,27 +32,40 @@ This article helps you understand the workload management and automation capabil
...
 
+- Review and implement [Cloud Adoption Framework costing best practices](https://learn.microsoft.com/azure/cloud-adoption-framework/govern/cost-management/best-practices).
+- Review and implement [Azure Well-Architected Framework cost optimization guidance](https://learn.microsoft.com/azure/well-architected/cost/overview).
+- Review and implement [Azure Advisor cost recommendations](https://learn.microsoft.com/azure/advisor/advisor-reference-cost-recommendations).
+  - Azure Advisor gives you high-confidence recommendations based on your usage. Azure Advisor is always the best place to start when looking to optimize any workload.
+  - Consider [subscribing to Azure Advisor alerts](https://learn.microsoft.com/azure/advisor/advisor-alerts-portal) to get notified when there are new cost recommendations.
```
Comment (flanakin) [inline docs/_docs/framework/capabilities/optimize/workloads.tobedeleted:56]:
```suggestion
- Review and implement [Azure Advisor cost recommendations](https://learn.microsoft.com/azure/advisor/advisor-reference-cost-recommendations).
  - Azure Advisor gives you high-confidence recommendations based on your usage. Azure Advisor is always the best place to start when looking to optimize any workload.
  - Consider [subscribing to Azure Advisor alerts](https://learn.microsoft.com/azure/advisor/advisor-alerts-portal) to get notified when there are new cost recommendations.
- Review and implement [Cloud Adoption Framework costing best practices](https://learn.microsoft.com/azure/cloud-adoption-framework/govern/cost-management/best-practices).
- Review and implement [Azure Well-Architected Framework cost optimization guidance](https://learn.microsoft.com/azure/well-architected/cost/overview).
```

File: docs/_docs/framework/capabilities/optimize/workloads.tobedeleted [hunk 16]
```diff
@@ -78,25 +91,25 @@ At this point, you have setup autoscaling and autostop behaviors. As you move be
...
+- Consider deploying other tools to help you optimizing your environment, for example the [Azure Optimization Engine]() available on FinOps toolkit provided by Microsoft.
 
 <br>
 
 ## 🍎 Learn more at the FinOps Foundation
 
-This capability is a part of the FinOps Framework by the FinOps Foundation, a non-profit organization dedicated to advancing cloud cost management and optimization. For more information about FinOps, including useful playbooks, training and certification programs, and more, see the [Workload management and automation capability](https://www.finops.org/framework/capabilities/workload-management-automation) article in the FinOps Framework documentation.
+This capability is a part of the FinOps Framework by the FinOps Foundation, a non-profit organization dedicated to advancing cloud cost management and optimization. For more information about FinOps, including useful playbooks, training and certification programs, and more, see the [Workload optimization capability](https://www.finops.org/framework/capabilities/workload-management-automation) article in the FinOps Framework documentation.
```
Comment (flanakin) [inline docs/_docs/framework/capabilities/optimize/workloads.tobedeleted:100]:
```suggestion
This capability is a part of the FinOps Framework by the FinOps Foundation, a non-profit organization dedicated to advancing cloud cost management and optimization. For more information about FinOps, including useful playbooks, training and certification programs, and more, see the [Workload optimization capability](https://www.finops.org/framework/capabilities/workload-optimization) article in the FinOps Framework documentation.
```

**Review by flanakin — APPROVED** [review]

**Pull request merged**